repository/managers.py

```diff
@@ -210,12 +210,20 @@ def _fetch_giles_text(self, giles_uploads):
             upload = giles_uploads[0]
             text_content = ""
 
-            # Extract plain text if available
+           # Extract plain text from Giles extracted text if available
             extracted_text = upload.get('extractedText', {})
-            if extracted_text and extracted_text.get('content-type') == 'text/plain':
+            if extracted_text and extracted_text['content-type'] == 'text/plain':
```

Comment:
is there a reason for this change? that will throw an exception if there is no property "content-type", won't it?